docs/v4-migration-doc.md

```diff
@@ -66,4 +71,7 @@ require additional action.
 
 ## Full list of changes
 
-:construction:
+- Drop Prometheus recording rule metrics
+
+  OpenTelemetry can't collect Prometheus recording rule metrics. The new version therefore stops collecting recording rule metrics and
+  updates will be made to the Kubernetes App to not show these metrics.
```

Comment:
I think it would help to provide a list of metrics that went away, and a list of Node metrics we collect instead.

Reply:
ok I'll mention this under the full list of changes

Reply:
its a long list :)
https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/e87dfe0a2f256df80e7677c4cd5500ac1b6fb366/docs/v4-migration-doc.md#full-list-of-changes